design/BufferOperations.md

```diff
@@ -152,25 +152,25 @@ stagingVertexBuffer.mapWriteAsync().then((stagingData) => {
 });
 ```
 
-### Updating data to an existing buffer (like webgl.bufferSubData)
+### Updating data to an existing buffer (like `gl.bufferSubData`)
 
-```
+```js
```

Comment:
Can you add `js` to AutoRingBuffer below too?

Reply:
Done.

Reply:
Done.